packages/renderer/src/App.svelte

```diff
@@ -79,86 +90,92 @@ window.events?.receive('display-help', () => {
       </div>
     </header>
 
-    <div class="overflow-x-hidden flex flex-1">
-      {#if meta.url.startsWith('/preferences')}
-        <PreferencesNavigation meta="{meta}" exitSettingsCallback="{() => router.goto(nonSettingsPage)}" />
-      {:else}
-        <AppNavigation meta="{meta}" />
-      {/if}
+    <Route path="/welcome">
+      <WelcomePage />
+    </Route>
 
-      <div class="z-0 w-full h-full bg-zinc-800 flex flex-col overflow-y-scroll">
-        <SendFeedback />
-        <ToastHandler />
-        <QuickPickInput />
-        <Route path="/">
-          <DashboardPage />
-        </Route>
-        <Route path="/containers">
-          <ContainerList searchTerm="{meta.query.filter || ''}" />
-        </Route>
-        <Route path="/containers/:id/*" let:meta>
-          <ContainerDetails containerID="{meta.params.id}" />
-        </Route>
+    {#if !meta.url.startsWith('/welcome')}
```

Comment:
hi, as welcome page is an overflow I'm not sure if we need to add the conditional if

Reply:
I think you don't need the `if` The issue is that the status bar is being displayed on top of your welcome screen ?
Should be just a z-axis  change on the welcome screen